src/cljs/main/broadfcui/page/groups/groups_management.cljs

```diff
@@ -109,7 +93,31 @@
      (endpoints/call-ajax-orch
       {:endpoint (endpoints/delete-group (:group-name @state))
        :on-done (net/handle-ajax-response
-                 #(swap! state assoc :delete-response %))}))})
+                 #(do
+                    (swap! state assoc :delete-response %)
+                    (this :-load-data))
+                )}))
```

Comment:
Avoid `#(do...`. Instead:
```clojure
(fn [response]
  (swap! state assoc :delete-response response)
  (this :-load-data))
```
Also, put these closing parens on the previous line.